### DISCUSSION TITLE
Questions about sharing and saving

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#154] [General]
Hi Kjell,
I have some questions emerging, but maybe I did not find if there is an help explaining that.
1) Where does Orbat mapper save the Local scenarios? is it possible to know the path where the files are locally stored (and see how they are done)?
2) Is it possible to save more than one scenario locally?
3) How is it possible to send a scenario to a friend for sharing?
Thank you in advance
Federico

**Comment by kjellmf** [comment]
I'm sorry for not documenting these things properly. 

1. When you use the "Save to local storage" the scenario is stored in you browser and not easily accessible outside of the browser. Currently you can only load it again using File -> Load scenario.
2. Currently you can only save one scenario to local storage. I will fix this in a future version. You can instead use File -> Download as JSON. This will download the scenario to your Downloads folder and you can load it again using File -> Load scenario
3. Yes. Either by downloading it as JSON and send it to them, or you can host the scenario file online somewhere and use the File -> Load scenario -> Enter url

On the landing page (https://orbat-mapper.app/) you also have some options for loading scenarios without using the File menu: 
![image](https://user-images.githubusercontent.com/29217/235129587-9290ef03-1b04-4243-b131-c62525d82141.png)

**Reply by federicopeyrani1969** [reply]
Thank you. 
I'll try and share my scenario.
Did you think about merging two scenarios? So I could work with a friend...
Best regards,
Federico

**Reply by kjellmf** [reply]
I will probably add some support for merging in the future. It should at least be possible to import parts of a scenario into another.

**Comment by kjellmf** [comment]
Most of this should be implemented now.
